enterprise/frontend/src/metabase-enterprise/collections/index.js

```diff
@@ -13,6 +13,11 @@ import {
 } from "./constants";
 
 PLUGIN_COLLECTIONS.REGULAR_COLLECTION = REGULAR_COLLECTION;
+
+PLUGIN_COLLECTIONS.isRegularCollection = collection =>
+  !collection.authority_level || // Root, personal collections don't have `authority_level`
+  collection.authority_level === REGULAR_COLLECTION.type;
+
```

Comment:
Nit…

We could make it shorter by destructuring `authority_level` as an arg.

Having the comments consistently above, and never after code, I'd say makes code more understandable.

```suggestion
PLUGIN_COLLECTIONS.isRegularCollection = ({authority_level}) =>
  // Root, personal collections don't have `authority_level`
  !authority_level || 
  authority_level === REGULAR_COLLECTION.type;

```

Again...total nitpicking.

Reply:
Also, it feels odd to me that we use arrow and traditional function declarations.

I'd personally prefer arrow function always, just my 2 cents.

Reply:
Good call, thank you! I'd suggest using function declarations as we discussed earlier. They're more explicit and it's easier to search for `function NAME(` than `const NAME = (`